src/test/java/uk/gov/companieshouse/api/accounts/validation/DebtorsValidatorTest.java

```diff
@@ -0,0 +1,287 @@
+package uk.gov.companieshouse.api.accounts.validation;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.function.Executable;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.companieshouse.api.accounts.exception.RestException;
+import uk.gov.companieshouse.api.accounts.model.rest.Accounts;
+import uk.gov.companieshouse.api.accounts.model.rest.CompanyProfile;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.Debtors.CurrentPeriod;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.Debtors.Debtors;
+import uk.gov.companieshouse.api.accounts.model.rest.notes.Debtors.PreviousPeriod;
+import uk.gov.companieshouse.api.accounts.model.validation.Error;
+import uk.gov.companieshouse.api.accounts.model.validation.Errors;
+import uk.gov.companieshouse.api.accounts.transaction.Transaction;
+import uk.gov.companieshouse.environment.EnvironmentReader;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class DebtorsValidatorTest {
+
+    private static String DEBTORS_PATH = "$.debtors";
+    private static String DEBTORS_PATH_PREVIOUS = DEBTORS_PATH + ".previous_period";
+
+    private static String CURRENT_TOTAL_PATH = DEBTORS_PATH + ".current_period.total";
+    private static String PREVIOUS_TOTAL_PATH = DEBTORS_PATH_PREVIOUS + ".total";
+    private static String PREVIOUS_TRADE_DEBTORS = DEBTORS_PATH_PREVIOUS + ".trade_debtors";
+
+    private Debtors debtors;
+    private Errors errors;
+
+    @Mock
+    private Transaction transaction;
```

Comment:
For ease of readability, could mocks prefix or suffix 'mock' - i.e `mockTransaction`, `mockRestTemplate` etc.